addons/account/migrations/12.0.1.1/post-migration.py

```diff
@@ -221,7 +221,28 @@ def set_default_taxes(env):
 
 
 def populate_fiscal_years(env):
-    if openupgrade.table_exists(env.cr, 'account_fiscalyear'):
```

Comment:
Better to keep this test, and if is false test if date_rangetype.

Otherwise. Lgtm. Thanks.

Reply:
Done